test/stream-deserialize.test.js

```diff
@@ -180,14 +182,14 @@ test('de/serialize: round-trip', function(t) {
     });
 
     function makeAssertions() {
-        var originalStats = fs.statSync(filepath);
-        var serializedStats = fs.statSync(tmpSerial);
-        var finalStats = fs.statSync(tmpDst);
-
-        var sizeDiff = Math.abs(originalStats.size - finalStats.size) / originalStats.size;
-
-        t.ok(sizeDiff < 0.01, 'round-tripped mbtiles are approx. the same size');
-        t.end();
+        before.getInfo(function(err, beforeInfo) {
+          t.ifError(err);
+          after.source.getInfo(function(err, afterInfo) {
+            t.ifError(err);
+            t.deepEqual(beforeInfo, afterInfo, 'input and output info is the same');
+            t.end();
+          });
+        });
```

Comment:
@rclark what do you think of this test instead of the approximate size test? Asserting that `getInfo` for the before and after MBTiles object is equal.